DBM-GUI/DBM-GUI_DropDown.lua

```diff
@@ -291,6 +291,7 @@ do
 					width = math.max(width, _G[dropdown:GetName().."Text"]:GetStringWidth())
 				end
 			end
+			if width > 300 then width = 300 end  -- Limit the width in case string has high width
```

Comment:
```suggestion
			if width > 400 then width = 400 end  -- Limit the width in case string has high width
```
just to be a bit more flush with the right border.

Also is a temp fix until proper area handling and window resizing is implemented in the GUI rewrite